generators/client/templates/angular/src/main/webapp/app/admin/admin.module.ts.ejs

```diff
@@ -64,8 +68,22 @@ import {
     <%_ } _%>
 } from './';
 
+export function translatePartialLoader(http: HttpClient) {
```

Comment:
@pascalgrimaud @ruddell @wmarques @jdubois, @mraible  FYI, this is fixing the missing translation issue from lazy load but its not ideal. This is why I wasn't originally a fan of merging the jsons, may be we should split them by module

@vishal423 did you take a look at the merge json webpack plugin we use, it shouldn't be hard to split them my module. Else lets make this piece reusable as it would be needed in all lazy modules soon

Reply:
@deepu105,
yes, I had look at merge plugin and also initially thought to split out. Few points related to TODO items:
- Few JSON keys specified in admin related bundles are shared between main and lazy loaded module. Before we split, we should move shared keys to common bundle so that they are available in main application and turn isolation off for lazy bundle to access main bundle keys (haven't tried, but, should work).
- To ensure reuse, can we move this as service to ng-jhipster project (which already have similar code for main)?

Reply:
Yes for both

Reply:
I have incorporated changes for 2nd point. IMO, factory doesn't make sense as dependencies are static per module.

Also, noted another issue with changeLanguage method as that doesn't affect lazy loaded modules :(

Reply:
@deepu105 
At moment, I could only reduce i18n bundle size of admin module. In case of main chunk, it still load all bundles as there is 'title' key dependency on sub-module bundles.

I also noticed that we have each admin sub-module key under global.json that's being used for navbar menu items. If we can use same key for page title, then, it will become much simpler. WDYT?